packages/dev/core/src/Physics/v2/Plugins/havokPlugin.ts

```diff
@@ -1389,8 +1389,10 @@ export class HavokPlugin implements IPhysicsEnginePluginV2 {
             return;
         }
 
+        const pluginDataArray = constraint._pluginData ?? [];
         const jointId = this._hknp.HP_Constraint_Create()[1];
-        constraint._pluginData = jointId;
+        pluginDataArray.push(jointId);
+        constraint._pluginData = pluginDataArray;

```

Comment:
nit - just wondering why this assignment is needed :-)

could be:

```
        constraint._pluginData = constraint._pluginData ?? [];
        const jointId = this._hknp.HP_Constraint_Create()[1];
        constraint._pluginData.push(jointId);
```

as you are using `constraint._pluginData` afterwards anyhow